src/main/java/com/google/api/codegen/util/SymbolTable.java

```diff
@@ -89,4 +114,24 @@ private String getAndSaveSuffix(String desiredName) {
     symbolTable.add(desiredName + Integer.toString(i));
     return Integer.toString(i);
   }
+
+  /**
+   * Returns the next underscore suffix that makes desiredName unique.
+   *
+   * <p>Stores the joined desiredName/suffix in an internal map. Assumes that desiredName is already
+   * in the symbolTable.
+   *
+   * <p>For example, if "foo" is passed, "_" is returned. If "foo" is passed again, "__" is
```

Comment:
This is acceptable in some languages? Having more than one trailing underscore seems so much more unreadable than just using numbers a la `foo3` or `foo_3`.